src/modules/drawing/components/SharePopup.vue

```diff
@@ -112,5 +147,9 @@ export default {
 .ga-share {
     width: 100%;
     text-align: start;
+
+    .input-group {
+        width: 30rem;
+    }
```

Comment:
as this is a Bootstrap class, I would maybe prefer that we add one custom class to the element you want to style this way, so that we do not overwrite Bootstrap's styling inadvertently (right now the style tag is scoped, so no big deal, but if someone were to remove the scoped flag, all Bootstrap `input-group` would be modified by this line)

Reply:
Good point.